node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -463,6 +463,12 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       descr = "the number of the active validators"
     )
 
+    val posVaultPubKey = opt[String](
+      descr = "Public key for transfers from the PoS vault. " +
+        "Base16 encoding of the public key of the PoS vault owner.",
```

Comment:
PoS vault is "owned" by unforgeable name and not by public key. The public key allows making transfers.

We can leave just the first sentence.